CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ if(MSVC)
 
 	# Silence Visual Studio error C2220
 	add_definitions(-D_SILENCE_STDEXT_ARR_ITERS_DEPRECATION_WARNING)
+	
+	add_compile_options(/bigobj) # TODO: [Georgii] remove this after fixing the issue with MNAEigenvalueExtractor.cpp (error C1128)
```

Comment:
`add_compile_options(/bigobj)` is already a fix suggested to the [error C1128](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/fatal-error-c1128?view=msvc-170) in the error message. This compile option is only applied to MSVC compiler. However, I see it as a workaround and not a nice solution.

I've started getting the error since [Make MNAEigenvalueExtractor a template class ](https://github.com/sogno-platform/dpsim/pull/262/commits/d5252bf07240b07aef967fc67327eea7fad6dc04) commit.
I've tried refactoring the `MNAEigenvalueExtractor` to simplify and reduce the number of functions in the class:

- [Refactor MNAEigenvalueExtractor](https://github.com/sogno-platform/dpsim/pull/262/commits/0803fd50bb73865698d402035fd33afa1b7f6d6d)
- [Remove specialized implementations of logging functions in MNAEigenvalueExtractor](https://github.com/sogno-platform/dpsim/pull/262/commits/adb2cd3dd2743013bba61a64078da7593084d08f)
- [Move eigenvalue logging to a separate class](https://github.com/sogno-platform/dpsim/pull/262/commits/751bfdf4477c23cef42743a24533639d6ce49e0b)

Further refactoring the `MNAEigenvalueExtractor`, e.g, reducing the number of functions or using two separate classes instead of a template class may help resolving the error without `add_compile_options(/bigobj)`. 

My idea is every time I refactor `MNAEigenvalueExtractor` to remove `add_compile_options(/bigobj)` and check if I still get the [error C1128](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/fatal-error-c1128?view=msvc-170). However, it looks like this TODO is confusing.

I suggest that I replace a TODO with a comment explaining the reason to have `add_compile_options(/bigobj)` :
`# Fixes error C1128 for compiling MNAEigenvalueExtractor with MSVC compiler`

What do you say?